storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -9,15 +9,22 @@ toc::[]
 
 {product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic Block Store (EBS).
 
+:FeatureName: AWS EBS CSI Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
 Familiarity with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs], link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent volume claims (PVCs)], link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
 provisioning], and link:https://kubernetes.io/docs/admin/authorization/rbac/[RBAC authorization] is recommended.
 
-Before PVCs can be created, you must install the AWS EBS CSI Driver Operator. The Operator provides a default StorageClass that you can use to create PVCs. You also have the option to create the EBS StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent Storage Using AWS Elastic Block Store].
+To create CSI-provisioned PVs that mount to AWS EBS storage assets, {product-title} installs the AWS EBS CSI Driver Operator and the AWS EBS CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
 
-After the Operator is installed, you must also create the AWS EBS CSI custom resource (CR) that is required in the {product-title} cluster.
+* The **AWS EBS CSI Driver Operator** provides a StorageClass by default that you can use to create PVCs. You also have the option to create the AWS EBS StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent Storage Using AWS Elastic Block Store].
 
-:FeatureName: AWS EBS CSI Driver Operator
-include::modules/technology-preview.adoc[leveloffset=+1]
+* The **AWS EBS CSI driver** enables you to create and mount AWS EBS PVs.
+
+[NOTE]
+====
+If you installed the AWS EBS CSI Operator and driver on a {product-title} 4.5 cluster, you must uninstall the 4.5 Operator and driver before you update to {product-title} {product-version}.
```

Comment:
Might be fine, but just want to make sure that this 4.5 hard code and the {product-version} is going to be okay going forward in the next version. For example, will this be okay to read as "if you installed on 4.5, you have to uninstall before updating to 4.7"? 

If not, then I would consider hard coding {product-version} to 4.6. We always do a search in releases for the current version number to see what needs to be updated, and at least this would come to our attention.

Or if you have plans to just delete this note completely for 4.7 and are tracking that, then it's fine too.

Reply:
Yes, this 4.5 hardcode and `{product-version}` attribute placement are correct. The 4.5 Operator version is available only in 4.5. So this note will apply only to users who: a) have installed the 4.5 Operator, and b) want to upgrade to a 4.6 or newer version.